requirements/requirements_ansible.txt

```diff
@@ -40,6 +40,7 @@ colorama==0.3.9           # via azure-cli-core, knack
 cryptography==2.1.4       # via adal, azure-storage, paramiko, pyopenssl, requests-kerberos, requests-ntlm, secretstorage
 decorator==4.2.1          # via openstacksdk
 deprecation==2.0          # via openstacksdk
+dnspython==1.15.0         # via dig
```

Comment:
This line should also be added to `requirements_ansible.in`, otherwise it will get wiped out the next time anyone runs `pip-compile requirements_ansible.in`.

Reply:
Fixed :-)